ipaserver/install/opendnssecinstance.py

```diff
@@ -274,6 +279,11 @@ def __setup_dnssec(self):
         if not self.fstore.has_file(paths.OPENDNSSEC_KASP_DB):
             self.fstore.backup_file(paths.OPENDNSSEC_KASP_DB)
 
+        if paths.ODS_ENFORCER is not None:
+            ods_cmd = paths.ODS_ENFORCER
+        else:
+            ods_cmd = paths.ODS_KSMUTIL
+
```

Comment:
Not to make things more complex but wouldn't it be better to put this difference into ipaplatform and use a generic name like ODS_MANAGER or something?

Reply:
Excellent idea! I move the calls into ``ipaplatform.tasks``. At first I had separate code for Debian and Fedora. But then I came to the conclusion that Fedora will eventually update to OpenDNSSEC 2.0, too. Now the abstraction is in the base tasks module.